src/js/components/skeleton.js

```diff
@@ -0,0 +1,15 @@
+const $skeletons = () =>
+  Array.from({ length: 10 }, () => 0)
+    .map(
+      () => `
+<div class="skeleton">
+  <div class="image"></div>
+    <p class="line"></p>
+    <p class="line"></p>
+    <p class="line"></p>
+  <div class="save"></div>
+</div>`
+    )
+    .join('');
+
+export default $skeletons;
```

Comment:
이렇게하면 $skeletons 불러올 때마다 함수가 실행될 것같은데, 아예 함수를 실행한 결괏값을 변수에 저장해놓고, 그 변수를 반환해주시는건 어떨까용 ??? 
```suggestion
const skeletonGenerator= () =>
  Array.from({ length: 10 }, () => 0)
    .map(
      () => `
<div class="skeleton">
  <div class="image"></div>
    <p class="line"></p>
    <p class="line"></p>
    <p class="line"></p>
  <div class="save"></div>
</div>`
    )
    .join('');

const $skeletons = skeletonGenerator();

export default $skeletons;
```

Reply:
동적으로 변경되는게 아니니 이게 성능면으로 훨씬 좋을것 같네요! 리뷰 넘 잘해주셔서 감사합니다...